doc/src/tutorials/physics/quantum_mechanics/creation_annihilation_tutorial.rst

```diff
@@ -0,0 +1,39 @@
+.. _creation_annihilation_tutorial:
```

Comment:
Please add these under `doc/src/tutorials/physics/quantum` to mirror the directory structure and names of the modules/packages in sympy.